terraform/api/k8s/main.tf

```diff
@@ -268,7 +268,7 @@ resource "kubernetes_service" "api" {
   }
 }
 
-resource "kubernetes_ingress" "api" {
+resource "kubernetes_ingress_v1" "api" {
```

Comment:
If we do this type of change while also updating to k8s 1.22, terraform will fail to deleted the previous ingress resource bc the resource spec was removed in 1.22